pipeline/restart_node.sh

```diff
@@ -116,6 +120,16 @@ shift 1
 
 log_define -v restart_node_log_level -l INFO rn
 
+echo "profile: ${HMY_PROFILE}"
```

Comment:
do you force to pass '-p' option?  you'd force to pass in '-p' command option.

Reply:
checked for a profile so that one has to set it to run the rest of the script